query/reverse.js

```diff
@@ -9,7 +9,12 @@ function generate( params ){
     lon: params.lon
   };
 
-  var query  =  queries.distance( centroid, { size: params.size || 1 } );
+  var query = queries.distance( centroid, {
+    size: params.size || 1,
+    sort: true,
+    distance: '500km'
```

Comment:
re: @harish's question about search radius:

Imagine the use case we're dealing with in Open by Mapzen, where we have a
mobile app that's being used (heavily) for driving directions. And it uses
today's geobias, which limits to 50km.


Great for getting to somewhere nearby (a school across town, a new
restaurant), but for longer trips (e.g. road trips to a different
province), that 50km radius won't work. 200's an attempt to patch that. It
doesn't solve for the magical universal geocoder that prefers places
closer, but still has a great global option for when the local places
aren't really matching (e.g starting in NYC and looking for an address in
Ottawa) but it's a start.

@insertcoffee - sound about right?



On Mon, Jul 13, 2015 at 11:31 AM, Harish Krishna <notifications@github.com>
wrote:

> In query/reverse.js
> <https://github.com/pelias/api/pull/156#discussion_r34475295>:
>
> > @@ -9,7 +9,12 @@ function generate( params ){
> >      lon: params.lon
> >    };
> >
> > -  var query  =  queries.distance( centroid, { size: params.size || 1 } );
> > +  var query = queries.distance( centroid, {
> > +    size: params.size || 1,
> > +    sort: true,
> > +    distance: '500km'
>
> why increase search radius?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pelias/api/pull/156/files#r34475295>.
>



-- 
David Riordan | Product Manager - Search | dave.riordan@mapzen.com |
@riordan <https://twitter.com/riordan> | gpg 235D9DC95EF6277C
<https://keybase.io/riordan>
Mapzen | https://mapzen.com | @mapzen <https://twitter.com/search>